### PR TITLE
[OBSDEF-10105] Add cpu limit and request for iam and federation

### PR DESCRIPTION
**Pull request by voa808**
## Purpose
[OBSDEF-10105] Add cpu limit and request for iam and federation to make the pods Guaranteed.
PR: Created for beta3 https://github.com/EMCECS/charts/pull/969

Testing: https://jira.cec.lab.emc.com/browse/OBSDEF-10105?focusedCommentId=6781457&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6781457

_List the changes for this PR_

## PR checklist
- [ ] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
https://asd-ecs-jenkins.isus.emc.com/view/ECS-Flex/job/charts-custom-ci/320/

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: logging-injector/Chart.yaml [hunk 1]
```diff
@@ -7,6 +7,6 @@ description: Rsyslog client sidecar injector
 engine: gotpl
 dependencies:
   - name: logging-injector
-    repository: https://asdrepo.isus.emc.com/artifactory/objectscale-helm-build/
+    repository: http://big-bang.lss.emc.com/export/home/polyav/build/charts/
```
[the remark this answers is not in the document]
Comment (lipak2345) [reply logging-injector/Chart.yaml:10]: @voa808 has Victor merged his changes? Can this be reverted now?

**Comment by lipak2345** [comment]
Where will the rsyslog side-car container resource requests be specified?

File: logging-injector/Chart.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (voa808) [reply logging-injector/Chart.yaml:10]: Talked to @victor-polyansky his changes are merged ~30 mins ago. I will revert the changes but will have to wait for charts to be built in order to run charts-ci for this branch 
`Victor Polyansky  1:55 PM
Hi Chetna, changes are merged, but charts are building now. Should be ready in ~3 hours.`

**Review by victor-polyansky — APPROVED** [review]

**Review by lipak2345 — APPROVED** [review]

**Review by eddiepavkovic — APPROVED** [review]
G2g.   Please indicate in the checklist it passed

**Pull request merged**